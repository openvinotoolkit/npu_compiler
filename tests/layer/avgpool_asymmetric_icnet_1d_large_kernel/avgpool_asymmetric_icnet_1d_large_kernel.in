version https://git-lfs.github.com/spec/v1
oid sha256:5fb7104e7397100d12f6761786c64f456e87d6364e7a0c6eb186beb7eb2e9e0e
size 1413120
