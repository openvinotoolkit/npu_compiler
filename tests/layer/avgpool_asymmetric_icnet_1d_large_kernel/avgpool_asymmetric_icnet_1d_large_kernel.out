version https://git-lfs.github.com/spec/v1
oid sha256:f62c1ec4681a0a1535dfb0539166e87770a59bc52bc479d30f88ce89a169bc0e
size 8192
